src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp

```diff
@@ -61,5 +61,56 @@ class OPENVINO_API ExperimentalDetectronGenerateProposalsSingleImage : public Op
     Attributes m_attrs;
 };
 }  // namespace v6
+
+namespace v8 {
+/// \brief An operation ExperimentalDetectronGenerateProposalsSingleImage
+/// computes ROIs and their scores based on input data.
+class OPENVINO_API ExperimentalDetectronGenerateProposalsSingleImage : public Op {
+public:
+    OPENVINO_OP("ExperimentalDetectronGenerateProposalsSingleImage", "opset8", op::Op, 8);
+    BWDCMP_RTTI_DECLARATION;
+
+    /// \brief Structure that specifies attributes of the operation
+    struct Attributes {
+        // minimum box width & height
+        float min_size;
+        // specifies NMS threshold
+        float nms_threshold;
+        // number of top-n proposals after NMS
+        int64_t post_nms_count;
+        // number of top-n proposals before NMS
+        int64_t pre_nms_count;
+        // specifies whether the output is dynamic shape or static shape
+        bool dynamic_output = false;
```

Comment:
Why do you need this "dynamic_output" attribute？

Reply:
this is previous design. will change it after the review of spec